bofire/data_models/features/continuous.py

```diff
@@ -19,6 +19,9 @@ class ContinuousInput(NumericalInput):
         stepsize (PositiveFloat, optional): Float indicating the allowed stepsize between lower and upper. Defaults to None.
         local_relative_bounds (Tuple[float, float], optional): A tuple that stores the lower and upper bounds relative to a reference value.
             Defaults to None.
+        allow_zero (bool): A boolean indicating if the input feature can take inactive values. If True,
+            use a conditional encoding kernel for this feature. Useful for features that take values between `bounds`, but can
```

Comment:
I would write that one can use a conditional kernel for this, but that it is not mandatory.